node_modules/oae-principals/lib/test/util.js

```diff
@@ -1047,3 +1125,256 @@ var assertUpdateUserFails = module.exports.assertUpdateUserFails = function(rest
         return callback();
     });
 };
+
+/**
+ * Assert that an email address can be verified
+ *
+ * @param  {RestContext}    restCtx         The REST context of a user who can perform the request
+ * @param  {String}         email           The email address to verify
+ * @param  {String}         token           The token that can be used to verify the email address
+ * @param  {Function}       callback        Invoked when the verify operation succeeds
+ * @param  {Object}         callback.err    An error that occurred, if any
+ * @throws {AssertionError}                 Thrown if the operation is unsuccessful or the expected results of verifying an email don't hold true
+ */
+var assertVerifyEmailSucceeds = module.exports.assertVerifyEmailSucceeds = function(restCtx, userId, token, callback) {
+    // Get the profile for the user before we verify their email address. This will allow
+    // us to assert that a new email address has been persisted
+    RestAPI.User.getUser(restCtx, userId, function(err, user) {
+        assert.ok(!err);
+        var oldEmailAddress = user.email;
+
+        // Verify the user has a pending email token
+        assertHasPendingEmailTokenSucceeds(restCtx, userId, function(email) {
```

Comment:
FWIW `assertHasPendingEmailToken`, or `assertPendingEmailTokenNotEmpty` would probably be both OK. There are similar patterns elsewhere with the name `assertSomethingEquals` as well. `Succeeds` is generally used when you just want to perform an operation (like create a user), and ensure that the operation was successful in some form.